packages/create-site/bin/mosaic-create-site.js

```diff
@@ -43,9 +43,6 @@ async function readMosaicConfig(mosaicConfigPath) {
     await init({ silent: true });
   }
   try {
-    const nodeModulesPathRegexp = /(.*\/node_modules\/)/;
-    const nodeModulesPathMatches = __filename.match(nodeModulesPathRegexp);
-    process.env.NODE_PATH = `${process.env.NODE_PATH}${path.delimiter}${nodeModulesPathMatches[1]}`;
```

Comment:
@mark-tate @joshwooding 

I don't know why we are doing this...can you shed any light?

It breaks the site generator on windows and removing seems to get everything working on mac and windows

Reply:
I quickly tried commenting these line in npx install location, it will not be able to find mosaicConfig